src/app/base/components/node/OverviewCard/DetailsCard/DetailsCard.test.tsx

```diff
@@ -44,48 +41,28 @@ beforeEach(() => {
   });
 });
 
-it("renders the domain", () => {
-  const machine = machineDetailsFactory({ domain: { id: 1, name: "maas" } });
-  state.machine.items = [machine];
-
-  const store = mockStore(state);
-  const wrapper = mount(
-    <Provider store={store}>
-      <MemoryRouter
-        initialEntries={[{ pathname: "/machine/abc123", key: "testKey" }]}
-      >
-        <CompatRouter>
-          <DetailsCard node={machine} />
-        </CompatRouter>
-      </MemoryRouter>
-    </Provider>
-  );
-
-  expect(wrapper.find("[data-testid='domain']").text()).toEqual("maas");
-});
-
 it("renders a link to zone configuration with edit permissions", () => {
   const machine = machineDetailsFactory({
     permissions: ["edit"],
     zone: { id: 1, name: "danger" },
   });
   state.machine.items = [machine];
 
-  const store = mockStore(state);
-  const wrapper = mount(
-    <Provider store={store}>
-      <MemoryRouter
-        initialEntries={[{ pathname: "/machine/abc123", key: "testKey" }]}
-      >
-        <CompatRouter>
-          <DetailsCard node={machine} />
-        </CompatRouter>
-      </MemoryRouter>
-    </Provider>
+  renderWithMockStore(
+    <MemoryRouter
```

Comment:
If the router is required then you could try with `renderWithBrowserRouter` instead of including the router here and if they don't work with either we should try and figure out if we need to adapt our render methods to make these work.

Reply:
Huh, I actually don't know why I didn't do that. I think it was force of habit from another test I was doing that didn't need a router